herast/tree/consts.py

```diff
@@ -20,7 +20,11 @@
 
 unary_expressions_ops = [
 	idaapi.cot_fneg, idaapi.cot_neg, idaapi.cot_cast, idaapi.cot_lnot, idaapi.cot_bnot, idaapi.cot_ptr,
-	 idaapi.cot_ref, idaapi.cot_postinc, idaapi.cot_postdec, idaapi.cot_preinc, idaapi.cot_predec, idaapi.cot_sizeof
+	idaapi.cot_ref, idaapi.cot_postinc, idaapi.cot_postdec, idaapi.cot_preinc, idaapi.cot_predec, idaapi.cot_sizeof
```

Comment:
yeah, there is indentation error, but is not a critical one :) usually such things (like spelling) are ignored, since fixing them break various git history things (like git blame)

Reply:
no problem, just noticed this inconsistency